### DISCUSSION TITLE
How about Replace react-i18next with next-i18next?

### DISCUSSION THREAD

**Discussion by winner106** [BearStudio/start-ui-web#137] [Ideas]
How about Replace react-i18next with next-i18next?

https://github.com/isaachinman/next-i18next

**Comment by yoannfleurydev** [comment]
Hi @winner106 , that's a good question, do you have any use cases to share with us that might let us consider this change?

`react-i18next` is working great for this starter at the moment, everything is setup to work out of the box. A replacement like that might take time and as I can see in the documentation, won't add feature to the project. Do you have any feature that is missing with `react-i18next`?

Also, 🚀 Start UI [web] is using Next.js at the moment, but this might change in the future, and setting up a dependency that is too tied up with Next.js is not that great in my opinion.

Let me know if you have any question, or benefits / drawbacks using any of the two i18n dependencies, I'm eager to learn more about your use cases.

**Reply by winner106** [reply]
Thanks for your reply, I got the answer from it.

**Comment by ivan-dalmet** [comment]
We've tried next-i18next but it's tied to next js url based translations. (/fr/app, /ar/app).
That fine for a website to have url based translations. But for an application we don't want this behavior. If you update your language in your account all urls should stay the same.

**Reply by yoannfleurydev** [reply]
I wasn't sure if you tested it, thanks for your feedback!

**Reply by winner106** [reply]
Thanks for your reply, I got the answer from it.
